pages/dkp/konvoy/1.6/addons/helmv2-to-v3-migration/index.md

```diff
@@ -30,6 +30,27 @@ kubectl -n kube-system delete deployment tiller-deploy
 
 This command shuts down Tiller gracefully and removes all its resources.
 
+## Upgrading Prometheus
+
+If you have the prometheus addon enabled and are upgrading it, you need to take additional steps to ensure it deploys successfully. You must delete the addon and then upgrade and deploy it.
+
+Before you begin the upgrade, run the following command to delete the addon:
+
+```bash
+kubectl delete addon prometheus -n kubeaddons
+helm2 delete --purge prometheus-kubeaddons
+```
+
+If you've already run the upgrade and deploying the prometheus addon has failed, run these commands:
+
+```bash
+kubectl delete addon prometheus -n kubeaddons
+helm2 delete --purge prometheus-kubeaddons
+konvoy deploy addons
```

Comment:
```suggestion
helm2 delete --purge prometheus-kubeaddons
```
This assumes a `helm2` cli tool being installed. Can we make this assumption?

Reply:
@joejulian 
I grabbed this from the conversation in Slack, so I haven't tested this scenario itself, but it seems like this worked for when Sergei ran it.

If this is the case, though, shoud it just be the same as your above suggestion
```
kubectl delete addon prometheus --namespace kubeaddons --wait
```
and then konvoy deploy addons?

Reply:
No, this is different because if they've already upgraded, deleting the addon will not delete the hold helm data.

Perhaps we should state that this expects them to have a helm 2 binary named `helm2`?

Reply:
Yeah, I think that's fair.

But, what if a user doesn't have helm2 installed anymore? Would that muck up the works?